runtime/j9vm/jvm.c

```diff
@@ -1150,6 +1150,7 @@ VersionSetting SHAPE_SETTINGS[] = {
 		{"b135", J2SE_SHAPE_B136},
 		{"b148", J2SE_SHAPE_B148},
 		{"b165", J2SE_SHAPE_B165},
+		{"b1803", J2SE_SHAPE_B1803},
```

Comment:
Should this be more specific? There were several _shapes_ for Java 9, we should assume there will be several for 18.3 as well.

Reply:
@pshipton indicated in https://github.com/eclipse/openj9/issues/152 "We expect to only support the latest so a build number should not be needed, and will quickly become out of date.".

Reply:
That makes sense. Thanks.